ssl/ssl_conf.c

```diff
@@ -656,6 +656,7 @@ static const ssl_conf_cmd_tbl ssl_conf_cmds[] = {
     SSL_CONF_CMD_SWITCH("no_renegotiation", 0),
     SSL_CONF_CMD_SWITCH("no_resumption_on_reneg", SSL_CONF_FLAG_SERVER),
     SSL_CONF_CMD_SWITCH("no_legacy_server_connect", SSL_CONF_FLAG_SERVER),
+    SSL_CONF_CMD_SWITCH("ignore_unexpected_eof", 0),
```

Comment:
This, and the other change in this file, should get removed

Reply:
Removing these changes causes an error when the ignore_unexpected_eof is provided via command line.

Reply:
To me that mean that it needs to be written in a different way.

Reply:
Done. Thanks!